internal/build/phases.go

```diff
@@ -215,15 +229,26 @@ func (l *Lifecycle) Build(ctx context.Context, networkMode string, volumes []str
 }
 
 func (l *Lifecycle) Export(ctx context.Context, repoName string, runImage string, publish bool, launchCacheName, cacheName, networkMode string, phaseFactory PhaseFactory) error {
-	export, err := l.newExport(repoName, runImage, publish, launchCacheName, cacheName, networkMode, phaseFactory)
+	var stackMount mount.Mount
+	if publish {
+		stackPath, err := l.writeStackToml()
```

Comment:
Bind mounting the stack.toml at `/cnb/stack.toml` (that is its current location). Another option that was discussed: copy the stack.toml to the `/layers` directory and explicitly pass its location to the lifecycle. I prefer this option as it seems closer to what we have now, but we could change it.

Reply:
Any downsides to this versus the alternative? I can't think of any but would like to confirm.

Reply:
Checking...

Reply:
Looking at the discussion here: https://github.com/buildpacks/lifecycle/pull/294#issuecomment-630388494 - it probably makes sense to copy the stack.toml into the layers directory (and set the env variable that tells the lifecycle where to look for it). Rationale: we'll have to do it this way for Tekton, it doesn't make sense to have two different approaches. But from a performance perspective the current approach is probably okay since the file is so small.

Reply:
Circling back - for now, we'll leave it as it is (bind mounting the stack.toml to its current location). We can explore changing the approach at a later date.